test/components/portal.spec.tsx

```diff
@@ -100,12 +108,14 @@ describe('<Portal />', function() {
     });
 
     it('renders with a className passed as a prop', async function() {
-        clientRenderer.render(
+        const {result} = clientRenderer.render(
             <Portal className="test-class">
                 <span data-automation-id="SPAN">Portal Body</span>
             </Portal>
         );
 
-        await waitFor(() => expect(bodySelect(portalId)!.className).to.contain('test-class'));
+        const driver = new PortalTestDriver(result as Portal);
+
+        await waitFor(() => expect((driver.portal as Element).className).to.contain('test-class'));
```

Comment:
have `classNames` getter for test driver

Reply:
Also, this test seems quite trivial. its being tested in the contract test no?

Reply:
This is here because portal is not contract tested. 
Also think `.className` is elegant enough to use over loading more getters into the driver.